src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -90,26 +108,51 @@ export const ConfirmBidRoute: React.FC<BidProps> = props => {
       errorMessages = [error.message]
     }
 
-    // TODO: add tracking with errorMessages
-    logger.error(errorMessages) // remove once tracking is implemented
+    trackConfirmBidFailed(bidderId, errorMessages)
 
     actions.setSubmitting(false)
     actions.setStatus("submissionFailed")
   }
 
+  function trackConfirmBidFailed(bidderId: string, errors: string[]) {
```

Comment:
thoughts on moving the `commonProperties` variable to be just above or below these track function definitions and rename to `commonTrackingProperties`? Personally, I think that would better clarify their intent in code.